www/login.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>CodeChecker</title>
+
+  <meta charset="UTF-8">
+
+  <link rel="shortcut icon" href="images/favicon.ico" />
+
+  <!-- CSS -->
+
+  <link type="text/css" rel="stylesheet" href="scripts/plugins/dojo/dijit/themes/claro/claro.css" />
+  <link type="text/css" rel="stylesheet" href="style/codecheckerviewer.css" />
+  <link type="text/css" rel="stylesheet" href="style/login.css" />
+  <link type="text/css" rel="stylesheet" href="style/productlist.css" />
```

Comment:
Why does the login use the productlist css?

Reply:
The styles for `formElement` and `form-input` are defined in the `productlist.css` file.